Send to Wikipedia sitelink with fallbacks via Hub

YoshiRulz opened the pull request:
Resolves #5.

> By default, the destination is Wikipedia in the user language, which is guessed from the request `accept-language` header, falling back to English if the language header can't be found or the Wikipedia page doesn't exist in this language. [...] Pass a `lang` parameter (or just `l`) to override the `accept-language` header. Pass several values to set the fallback chain. The value `auto` can be used to represent the value of the `accept-language` header.

&mdash;https://hub.toolforge.org/#destination

> By default, when a destination is not found, you are redirected to the Wikidata entity page.

&mdash;https://hub.toolforge.org/#fallback

YoshiRulz committed:
Send to Wikipedia sitelink with fallbacks via Hub

Jk40git reviewed (changes requested):
let's go according to the special:gotolinkedpage. 
One thing also we want the users to contribute to missing pages or non existent articles. Sending them to an English fallback pages doesn't promote that.

YoshiRulz commented:
It shouldn't go to enwiki if the languages set in the browser don't include `en` or `en-US`. (Though when I went to test it just now, Firefox wouldn't let me remove `en-US`.)

Jk40git committed:
Update popup.js

@YoshiRulz something like this will be fine cause it prevents the default fallback to English wikis

Jk40git commented:
Closing this Thanks @YoshiRulz for the contribution
